src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -31,6 +32,21 @@ public static boolean isValidPath(String path) {
         return true;
     }
 
+    //@@author jitwei98
+    /**
+     * Returns true if {@code filename} is a valid filename for xml files,
+     * otherwise returns false.
+     * @param filename A string representing the file path. Cannot be null.
+     */
+    public static boolean isValidXmlFilename(String filename) {
+        if (filename.length() <= XML_FILE_EXTENSION.length()) {
+            return false;
+        }
+
+        // Compares the filename extension with the expected xml file extension
+        return filename.substring(filename.length() - XML_FILE_EXTENSION.length()).equals(XML_FILE_EXTENSION);
```

Comment:
think this might violate LOD so you might want to refactor it

Reply:
Modified according to this.